qbft/state.go

```diff
@@ -28,22 +24,22 @@ type IConfig interface {
 	GetNetwork() Network
 	// GetTimer returns round timer
 	GetTimer() Timer
+	// GetSignatureVerifier returns the signature verifier for operator signatures
+	GetSignatureVerifier() types.SignatureVerifier
+	// GetRoundCutOff returns the round cut off
+	GetCutOffRound() int
```

Comment:
Why does it return `int` instead of `qbft.Round`?

Reply:
changed